src/renderer/colors.css

```diff
@@ -0,0 +1,53 @@
+:root {
+    /* color palette */
+    --gray-50: #f7fafc;
+    --gray-100: #edf2f7;
+    --gray-200: #e2e8f0;
+    --gray-300: #cbd5e0;
+    --gray-400: #a0aec0;
+    --gray-500: #718096;
+    --gray-600: #4a5568;
+    --gray-650: #3e485a;
+    --gray-700: #2d3748;
+    --gray-750: #252d3c;
+    --gray-800: #1a202c;
+    --gray-850: #191d28;
+    --gray-900: #171923;
+
+    /* backgrounds */
+    --bg-800: var(--gray-800);
+    --bg-700: var(--gray-700);
+    --bg-600: var(--gray-600);
+
+    /* node colors */
+    --node-border-color: var(--gray-800);
```

Comment:
Is there a point to having the word "color" in these?

Reply:
CSS variables can be anything. So `--something-border: 1px solid #FFF` is what I would expect from a border.

Reply:
that makes sense